lib/sproxyd.js

```diff
@@ -303,15 +304,18 @@ class SproxydClient {
 
             const request = _createRequest(req, log, (err, response) => {
                 if (err) {
+                    const errorProcessed = err;
                     if (streamingStarted || voluntaryAbort) {
-                        err.retryable = false;
+                        errorProcessed.retryable = false;
                     } else {
-                        err.retryable = true;
+                        errorProcessed.retryable = true;
                     }
                     if (!voluntaryAbort) {
-                        log.error('putting chunk to sproxyd', {error: err});
+                        log.error('putting chunk to sproxyd', {
+                            error: errorProcessed,
+                        });
                     }
-                    return callback(err);
+                    return callback(errorProcessed);
```

Comment:
If i'm not mistaken, this will have the same behaviour as previous code. `err` will still be modified as `errorProcessed` contains a reference to it and not a copy of its value. To copy the value we can use `const errorProcessed = New Error(err);`

Reply:
Yes, this was expected, because we would like to avoid the eslint error, saying that we directly modify a parameter, I think there is no need to recreate an error here, it is only code-style, to avoid ignoring the eslint rule